.github/workflows/ci.yml

```diff
@@ -167,12 +212,12 @@ jobs:
           # This is a bit of a hack, but cargo apk doesn't seem to allow customising this
           RUSTFLAGS: '-D warnings'
 
-  docs:
+  doc:
     name: cargo doc
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, macos-latest, ubuntu-latest]
+        os: [windows-latest, macos-14, ubuntu-latest]
```

Comment:
Is it necessary to document for all platforms here? We don't have any platform specific code at this level

Reply:
Yeah, the Vello repo doesn't have an active need for it. It doesn't increase wall time in most scenarios, but if there are a bunch of PRs doing CI at the same time, these extra jobs can actually start to matter. Thus I reduced the doc job to only Ubuntu here and added a note about potential future expansion.